### ISSUE TITLE
Screen readers are not announcing the name of the AI model generating responses in the GitHub Copilot chat: A11y_Visual Studio Code Client_ScreenReader

### ISSUE THREAD

**Issue by AnKushSingh05** [microsoft/vscode#283834]
### GitHub Tags:
#A11yTCS;#Visual Studio Code Client;#BM_VisualStudioCodeClient_Win32_JULY2024; #DesktopApp; #Win32; #Benchmark;#A11yUsable;

### Environment and OS details:
Application: VS Code
Version: 1.108.0-insider (user setup)

### Reproduction Steps:

1. Open Visual studio code insider.
2. Tab till copilot chat.
3. Ask something to copilot.
4. Observed that Screen readers are not announcing the name of the AI model generating responses in the GitHub Copilot chat.

### Actual Result:
Screen readers are not announcing the name of the AI model generating responses in the GitHub Copilot chat.

### Expected Result:
Before starting to read the response, screen readers should automatically announce the name of the AI model generating the response.
Alternatively, provide a setting to allow screen readers announce the name of the AI model generating the response.

### User Impact:
Screen reader users are unable to identify the AI model generating the response for the prompt.

### Attachment

https://github.com/user-attachments/assets/68fa4046-0b17-43be-9950-3a36e3e7a13e
